index.js

```diff
@@ -77,7 +77,7 @@ const sortObjectBySemver = sortObjectBy((a, b) => {
   const { name: bName, range: bRange } = parseNameAndVersionRange(b)
 
   if (aName !== bName) {
-    return aName.localeCompare(bName)
+    return aName > bName ? 1 : -1
```

Comment:
The change to use `localeCompare` came from https://github.com/keithamus/sort-package-json/pull/314 and seems well motivated. I understand that it's going to be dependant on the locale, which isn't great, but moving back to comparing character points seems sub-optimal. I wonder if fixing the locale would be a better choice?

```suggestion
    return aName.localeCompare(bName, 'en')
```

Reply:
This was my first idea, but I didn't want to hardcode one specific language. But it will work. I am fine with it, English is de facto the standard language in the programming world anyway.

Reply:
I agree that it feels bad to hard-code a particular language but in this instance it seems well justified. no?

Reply:
Yeah, I agree. I applied your suggestion.